### PR TITLE
[regen] No more into

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Not to be merged.

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by GuillaumeGomez** [comment]
cc @sdroege @EPashkin

**Comment by EPashkin** [comment]
Don't want change manual parts too?
Thanks and 👍 for this batch anyway

**Comment by GuillaumeGomez** [comment]
That's a good point! I'll go over the manual part as well.

File: src/auto/accel_label.rs [hunk 1]
```diff
@@ -53,9 +53,9 @@ pub trait AccelLabelExt: 'static {
 
     fn set_accel(&self, accelerator_key: u32, accelerator_mods: gdk::ModifierType);
 
-    fn set_accel_closure<'a, P: Into<Option<&'a glib::Closure>>>(&self, accel_closure: P);
+    fn set_accel_closure(&self, accel_closure: Option<&glib::Closure>);
 
-    fn set_accel_widget<'a, P: IsA<Widget> + 'a, Q: Into<Option<&'a P>>>(&self, accel_widget: Q);
+    fn set_accel_widget<P: IsA<Widget>>(&self, accel_widget: &P);
```
Comment (GuillaumeGomez) [inline src/auto/accel_label.rs]: That should be `Option<&P>`, not `&P`. The PR is invalid. :)

**Comment by GuillaumeGomez** [comment]
Updated!

File: src/auto/accel_label.rs (hunk 1 above)
Comment (EPashkin) [reply src/auto/accel_label.rs]: Hm, this still not fixed, seems update go wrong as it have right in my regen.

Comment (GuillaumeGomez) [reply src/auto/accel_label.rs]: I think my rebase went crazy. I'll restart the whole process.

**Comment by EPashkin** [comment]
Thanks, 👍

File: src/auto/about_dialog.rs [hunk 2]
```diff
@@ -472,97 +463,97 @@ impl<O: IsA<AboutDialog>> AboutDialogExt for O {
...
 where P: IsA<AboutDialog> {
-    let f: &F = transmute(f);
+    let f: &F = &*(f as *const F);
     f(&AboutDialog::from_glib_borrow(this).unsafe_cast())
 }
 
 unsafe extern "C" fn notify_comments_trampoline<P, F: Fn(&P) + 'static>(this: *mut ffi::GtkAboutDialog, _param_spec: glib_ffi::gpointer, f: glib_ffi::gpointer)
 where P: IsA<AboutDialog> {
-    let f: &F = transmute(f);
+    let f: &F = &*(f as *const F);
```
Comment (sdroege) [inline src/auto/about_dialog.rs]: You probably want to remove the `transmute` in the manual code too. There are 5 cases like this in the manual code that can be changed, leaving only the cases where the trampolines are transmuted (I'm not sure why exactly this is needed)

**Comment by sdroege** [comment]
> +1 except for the failing doctests

... and the remaining transmutes

**Comment by EPashkin** [comment]
👍

**Pull request merged**